cognite_toolkit/cdf_tk/load/_resource_loaders.py

```diff
@@ -207,7 +207,8 @@ def load_resource(
         for capability in raw.get("capabilities", []):
             for acl, values in capability.items():
                 scope = values.get("scope", {})
-                is_resource_scoped = any(scope_name in scope for scope_name in self.resource_scope_names)
+                if not is_resource_scoped:
+                    is_resource_scoped = any(scope_name in scope for scope_name in self.resource_scope_names)
```

Comment:
What does this solve? Any failing tests?

Reply:
Observed that if the last ACL in the list of capabilities of the group is "all", the is_resource_scope gets set to False. When you then get to this code further down, it doesn't matter that other ACLs had scopes:

```
if not is_resource_scoped and self.target_scopes == "resource_scoped_only":
            # If a group has no resource scoped capabilities, we skip it.
            return None
```
So the change is to ensure that if `is_resource_scope ` has been set to True once, we don't risk setting it back to False again.